xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -590,6 +596,46 @@ void CGUIDialogVideoInfo::ClearCastList()
   m_castList->Clear();
 }
 
+void CGUIDialogVideoInfo::OnChangeVersion()
+{
+  CVideoDatabase videodb;
+  if (!videodb.Open())
+    return;
+
+  CFileItemList list;
+  int dbId = m_movieItem->GetVideoInfoTag()->m_iDbId;
+
+  CGUIDialogSelect* dialog = CServiceBroker::GetGUI()->GetWindowManager().GetWindow<CGUIDialogSelect>(WINDOW_DIALOG_SELECT);
+  if (!dialog)
+    return;
+
+  int selected = -1;
+
+  videodb.GetTypesNav("videodb://movies/types", list, dbId);
```

Comment:
I would move `CFileItemList list;` and `int dbId = m_movieItem->GetVideoInfoTag()->m_iDbId;` above this line, where these vars are first needed.

Reply:
... and `videodb` decl plus open + check + ret as well.